clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -3125,6 +3129,10 @@ static void RenderFloatingPointOptions(const ToolChain &TC, const Driver &D,
       TrappingMathPresent = true;
       FPExceptionBehavior = "strict";
       break;
+    case options::OPT_fveclib:
+      if (llvm::is_contained(VecLibImpliesNoMathErrno, A->getValue()))
+        MathErrno = false;
+      break;
```

Comment:
Should we check specifically for the Arm platform?

Reply:
I don't thinks so, these 2 libraries are stating clearly that they do not honour errno regardless of the platform. Well in ArmPL case is supports only Arm, but SLEEF could work on other targets if the right mappings were added to LLVM.

Reply:
As @mgabka says, this is a property of the library rather than an architecture specific decision.